leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -275,14 +275,13 @@
   },
 
   CLIPBOARD_UI_MANAGER__SINSTANCE(
-      SAMSUNG.equals(MANUFACTURER) && SDK_INT >= KITKAT && SDK_INT <= LOLLIPOP) {
```

Comment:
Thanks. I have two questions though:

1) Are you sure that the leak didn't exist on Kitkat? The fact that the fix doesn't work on Kitkat doesn't mean that the leak doesn't exist. The fix should be gated, but not the leak.
2) Are you sure this leak still exists on M?